<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="windowTitle">
   <string>Crypto safe</string>
  </property>
  <property name="styleSheet">
   <string>
    background-color: #f4f8fb;
    border-radius: 16px;
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Cryptage &amp; Décryptage de Fichier</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string>
        font-size: 22px;
        color: #2d89ef;
        font-weight: bold;
        margin-bottom: 10px;
      </string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="fileLayout">
     <item>
      <widget class="QLineEdit" name="filePathEdit">
       <property name="placeholderText">
        <string>Chemin du fichier...</string>
       </property>
       <property name="styleSheet">
        <string>
          font-size: 16px;
          padding: 6px 10px;
          border: 2px solid #b0c4de;
          border-radius: 8px;
          background: #fff;
        </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browseButton">
       <property name="text">
        <string>Parcourir</string>
       </property>
       <property name="styleSheet">
        <string>
          background-color: #2d89ef;
          color: white;
          font-weight: bold;
          font-size: 15px;
          border-radius: 8px;
          padding: 6px 18px;
        </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="modeLayout">
     <item>
      <widget class="QRadioButton" name="encryptRadio">
       <property name="text">
        <string>Cryptage</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string>
          font-size: 15px;
          color: #2d89ef;
          font-weight: bold;
        </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="decryptRadio">
       <property name="text">
        <string>Décryptage</string>
       </property>
       <property name="styleSheet">
        <string>
          font-size: 15px;
          color: #e67e22;
          font-weight: bold;
        </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordEdit">
       <property name="placeholderText">
        <string>Mot de passe (décryptage)</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="styleSheet">
        <string>
          font-size: 16px;
          padding: 6px 10px;
          border: 2px red;
          border-radius: 8px;
          background: #fffbe6;
        </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="actionButton">
     <property name="text">
      <string>Exécuter</string>
     </property>
     <property name="styleSheet">
      <string>
        background-color: #27ae60;
        color: white;
        font-weight: bold;
        font-size: 18px;
        border-radius: 10px;
        padding: 10px 0;
        margin-top: 10px;
      </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Annuler</string>
     </property>
     <property name="styleSheet">
      <string>
        background-color: #e74c3c;
        color: white;
        font-weight: bold;
        font-size: 16px;
        border-radius: 8px;
        padding: 8px 0;
        margin-bottom: 6px;
      </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string>
        QProgressBar {
          border: 2px solid #b0c4de;
          border-radius: 8px;
          text-align: center;
          font-size: 15px;
          background: #fff;
        }
        QProgressBar::chunk {
          background-color: #2d89ef;
          border-radius: 8px;
        }
      </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <connections>
  <connection>
   <sender>actionButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_actionButton_clicked()</slot>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_cancelButton_clicked()</slot>
  </connection>
 </connections>
</ui>
